<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui::UserAgentPage</class>
 <widget class="QWidget" name="QtGui::UserAgentPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTTP user agent</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="defaultRadioButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Use a ra&amp;ndom default user agent</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="customRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Custom user agent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QtUtilities::ClearLineEdit" name="customLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>leave empty to use no user agent</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtUtilities::ClearLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">qtutilities/widgets/clearlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>customRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>customLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
